livingatlas/scripts/la-pipelines

```diff
@@ -22,7 +22,7 @@ fi
 
 # https://mikefarah.gitbook.io/yq/upgrading-from-v3
 YQ_VERSION="$(yq -V  | sed 's/.* //g')"
-if [[ $YQ_VERSION =~ ^3 ]]
+if [[ $YQ_VERSION =~ ^[a-zA-Z]*3\.[0-9]*\.[0-9]* ]]
```

Comment:
As this is something that started in some point of version 4 and adding a `v`, maybe we can let this part untouched (as was working with version 3).